pkgs/development/libraries/libcommuni/default.nix

```diff
@@ -0,0 +1,30 @@
+{ fetchgit, qt5, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  name = "libcommuni-${version}";
+  version = "779b0c7744";
```

Comment:
Please change version to date. See https://nixos.org/nixpkgs/manual/#sec-package-naming